client/resource_source.go

```diff
@@ -84,6 +84,12 @@ func ResourceSource() *schema.Resource {
 				MaxItems: 1,
 				Elem:     kafkaSourceDestinationSchema(),
 			},
+			"snowflake": {
+            	Type:     schema.TypeList,
```

Comment:
small one: just formatting

Reply:
ah!! thought i fixed up all of them..my bad..these ones appear correctly in intellij ..fixed now!